### ISSUE TITLE
Change default fonts to Source Family on all platforms please

### ISSUE THREAD

**Issue by XinoAssassin** [JoyNeop/yart#4]
Source Family Fonts are more professional in many ways compared to some platform built-in fonts. And it's free more users. You can use Google Fonts to offer the web fonts.

**Comment by neruthes** [comment]
Let's look into both major scenarios:

- Online Display
- PDF/Printed

For online displays, one should be able to customize typography and many things; this project is supposed to be just a framework which personalizations are based on. Therefore, the philosophy of the aesthetics of this project is to have as little visual design as possible. In this case, the developer is responsible for his/her own aesthetics of presentation, not me.

For PDF and printed, one may always export PDF on his/her own device instead of asking a reader to do this, in order to keep everything as intended as possible. Also notice that major browsers require a font to be installed locally to make it available for print and PDF exporting, so web fonts would not help.

In conclusion, I see no outcome overcoming the price to include web fonts, especially internationalization ones, by default.

Thank you.

**Comment by XinoAssassin** [comment]
Thanks for list the reasons.
I ignored the possibility that I can personalize the resume page. How the resume seems like is based on the users' own aesthetics of presentation.
